### ISSUE TITLE
As of this morning I'm getting an http 400 error for all calls to the api?!?

### ISSUE THREAD

**Issue by AlCaughey** [basecrm/basecrm-php#34]
no changes to the code at my end the following test file
```
<?php
require 'vendor/autoload.php';
$client = new \BaseCRM\Client(['accessToken'  => 'a-valid-token']);
$n=1;
$deals = $client->deals->all($options = ['per_page'=>100, 'page'=>$n]); 
echo "<!--";
print_r($deals);
echo "-->";
?>
```
results in 
```
<b>Fatal error</b>:  Uncaught BaseCRM\Errors\RequestError in C:\Apache24\htdocs\vendor\basecrm\basecrm-php\lib\HttpClient.php:304
--
  | Stack trace:
  | #0 C:\Apache24\htdocs\vendor\basecrm\basecrm-php\lib\HttpClient.php(286): BaseCRM\HttpClient-&gt;handleErrorResponse(400, '&lt;html&gt;\r\n&lt;head&gt;&lt;...', NULL)
  | #1 C:\Apache24\htdocs\vendor\basecrm\basecrm-php\lib\HttpClient.php(167): BaseCRM\HttpClient-&gt;handleResponse(400, '&lt;html&gt;\r\n&lt;head&gt;&lt;...')
  | #2 C:\Apache24\htdocs\vendor\basecrm\basecrm-php\lib\HttpClient.php(30): BaseCRM\HttpClient-&gt;request('get', '/deals?per_page...', Array, NULL, Array)
  | #3 C:\Apache24\htdocs\vendor\basecrm\basecrm-php\lib\DealsService.php(42): BaseCRM\HttpClient-&gt;get('/deals', Array)
  | #4 C:\Apache24\htdocs\test.php(14): BaseCRM\DealsService-&gt;all(Array)
  | #5 {main}
  | thrown in <b>C:\Apache24\htdocs\vendor\basecrm\basecrm-php\lib\HttpClient.php</b> on line <b>304</b><br />
```
Is anyone else seeing this?

**Comment by waingro** [comment]
We are having the same issue.
I was able to make a successful request in the PHPStorm's built-in http client.

**Comment by needls-mike** [comment]
Same issue for us.  We had to disable Base because it was breaking everything.  As mentioned by waingro, using insomnia/Postman worked so seems to be an issue with the library.

**Comment by AlCaughey** [comment]
I just got an email from Base... apparently they have been able to fix the
issue...

**Comment by xcartman** [comment]
https://github.com/basecrm/basecrm-php/issues/35

**Comment by AlCaughey** [comment]
Closing this issue as per the Base ticket:
https://support.getbase.com/hc/en-us/requests/154082

**Comment by efedasz** [comment]
Hi there.
We're sorry for the inconvenience. We've released a new version [v1.4.0](https://github.com/basecrm/basecrm-php/releases/tag/v1.4.0) with fixed HTTP method names.
We plan to upgrade our infrastructure, which can cause issues with older api-clients, so please upgrade your libraries.

Thank you!
